src/scripts/import-open-api.ts

```diff
@@ -0,0 +1,417 @@
+import { pascal } from "case";
+import get from "lodash/get";
+import groupBy from "lodash/groupBy";
+import isEmpty from "lodash/isEmpty";
+import uniq from "lodash/uniq";
+
+import {
+  ComponentsObject,
+  OpenAPIObject,
+  OperationObject,
+  ParameterObject,
+  PathItemObject,
+  ReferenceObject,
+  RequestBodyObject,
+  ResponseObject,
+  SchemaObject,
+} from "openapi3-ts";
+
+// @ts-ignore - no type definition here
```

Comment:
Can we polyfill with a `.d.ts` file and `declare module "swagger2openapi"`?

Reply:
I tried… It was just not working, but I will give another try